### PR TITLE
Small fixes for console display.

### PR DESCRIPTION
**Pull request by chadwcarlson**
- Resolves DEVREL-83: Updates primary tag for Hugo to Go
- Makes "Beego" reference consistent https://beego.me/docs/intro/
- Substitutes html tags for markdown syntax for Drupal 8 and Symfony4
- Resolves https://github.com/platformsh/template-symfony3/pull/11: Updates parameters_platformsh.php file reference.

### CODE REVIEW

**Comment by otaviojava** [comment]
I only saw the template-beego PR, please, don't forget to include the others.
The changes look fine to me.

**Comment by chadwcarlson** [comment]
Removed the Symfony3 change for now, I'll work on that in a separate PR since it's description still needs to be updated.

File: templates/hugo/.platform.template.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ info:
       <p>Hugo is a static site generator written in Go, using Go's native template packages for formatting.</p>
   # A list of tags associated with the template.
   tags:
+  - Go
   - Hugo
```
Comment (Crell) [inline templates/hugo/.platform.template.yaml]: We can probably remove the Hugo tag while we're at it, since there's nothing else with it.

**Comment by otaviojava** [comment]
@chadwcarlson that still several projects on it such as Payara, oracle Helidon, Drupal and so on.

**Comment by chadwcarlson** [comment]
@otaviojava Beego is the only change that affects its template repo, since .platform.template.yaml has been moved outside of files.

**Comment by chadwcarlson** [comment]
Well Hugo now too.

**Pull request merged**